scripts/create-unity-package.sh

```diff
@@ -10,17 +10,28 @@ done
 PROJECT="$PWD/Subtester"
 PACKAGE="$PWD/Purchases.unitypackage"
 PACKAGE_UNITY_IAP="$PWD/Purchases-UnityIAP.unitypackage"
+
+# To export a .unitypackage, we need to have the SDK files inside the Subtester project
+# Additionally, we need to remove the UPM dependency from the manifest to export it,
+# otherwise it will cause issues due to duplicated symbols.
+SYMBOLIC_LINK_PATH="$PROJECT/Assets/RevenueCat"
+MANIFEST_JSON_PATH="$PROJECT/Packages/manifest.json"
+ln -s "$PWD/RevenueCat" "$PROJECT/Assets/"
+awk '!/com.revenuecat.purchases-unity/' $MANIFEST_JSON_PATH > temp && mv temp $MANIFEST_JSON_PATH
```

Comment:
This is removing the UPM dependency correct?

Reply:
Yes, probably good to add a comment here explaining it. But basically, if we leave the UPM dependency here and we add the symlink to the RevenueCat folder, it will fail to create the `unitypackage` due to duplicate files, so we need to remove the UPM dependency as part of creating the `unitypackage`. Will add a comment explaining it.

Reply:
Oh actually, I had a comment above, I forgot 😅. Do you think we should make it clearer here?

Reply:
yeah, default to over-explaining for stuff like this, since this is not a trivial line. a maintainer first needs a min to understand what this even does and how, then another to figure out why. we need to save them as much time as possible.

Reply:
you know the old phrase "write code like the person maintaining it is a serial killer who knows where you live"